web/src/features/fireWeather/components/graphs/PrecipGraph.tsx

```diff
@@ -147,6 +184,26 @@ const PrecipGraph: React.FunctionComponent<Props> = ({
       }
     )
 
+    const accumObservedPrecips: { date: Date; accumPrecip: number }[] = []
+    observedPrecips.forEach(({ date, value }) => {
+      const observed = { date, accumPrecip: NaN }
+      if (value != null) {
+        if (accumObservedPrecips.length === 0) {
+          observed.accumPrecip = value
+        } else {
+          observed.accumPrecip =
+            value + accumObservedPrecips[accumObservedPrecips.length - 1].accumPrecip
+        }
+        accumObservedPrecips.push(observed)
+      }
+    })
+    if (
+      accumObservedPrecips.length > 0 &&
+      maxAccumPrecip < accumObservedPrecips[accumObservedPrecips.length - 1].accumPrecip
+    ) {
+      maxAccumPrecip = accumObservedPrecips[accumObservedPrecips.length - 1].accumPrecip
+    }
+
```

Comment:
Wouldn't always the last element of the array be the max? same goes for the forecast as well

Reply:
OH that's what you are doing lol. Nvm